A01-Extensions/A01-How_to_write_ASDF_extensions.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>should we talk about <code>pyproject.toml</code> here or is that too much information </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I figure we can just punt that to go look at the docs.